LIPs/LIP-36.md

```diff
@@ -31,6 +31,10 @@ Note: The mathematical explanation is included the Discourse forum post instead
 
 ### Data Types
 
+#### LIP_UPGRADE_ROUNDS 
+
+This is an newly introduced mapping that maps an LIP number to a round at which it has been introduced. This is helpful in case an LIP has breaking changes such as this one. This notion of an upgrade round allows us to switch between the old algorithm and new algorithm.
```

Comment:
Let's be specific about which algorithm we're referring to here since earlier in the paragraph we note that this mapping could be generally used for different scenarios with breaking changes.

Suggestion: "old earnings claiming algorithm and the new earnings claiming algorithm".

Reply:
fixed in 4823679